Sources/NIOCore/Utilities.swift

```diff
@@ -258,3 +258,18 @@ extension System {
     public static let supportsUDPReceiveOffload: Bool = false
     #endif
 }
+
+extension System {
```

Comment:
Wy is this in a new extension?

Reply:
Just for separation of groups of related code - but I'm happy to lump it together if you'd rather.

Reply:
I've consolidated them now that I'm only adding one thing.